tester/main.cpp

```diff
@@ -70,26 +61,34 @@ int main(int argc, char const *argv[]) {
 	vc::Option dataSize_medium("dataType", "-DMEDIUM_DATASET");
 	vc::Option dataSize_large("dataType", "-DLARGE_DATASET");
 	vc::Option dataSize_xlarge("dataType", "-DEXTRALARGE_DATASET");
+
+	const opt_vec_t size_vec = {
+		vc::make_option("-DMINI_DATASET"),
+		vc::make_option("-DSMALL_DATASET"),
+		vc::make_option("-DMEDIUM_DATASET"),
+		vc::make_option("-DLARGE_DATASET"),
+		vc::make_option("-DEXTRALARGE_DATASET"),
+	};
```

Comment:
When you start using the `vc::make_option()` within the vector initialization, the above variables `datasize_xxx` are no longer needed. I suggest to remove them.